tests/framework/rbac/BaseManagerTest.php

```
@@ -0,0 +1,351 @@
+        $this->mockApplication();
+
+        $runtimePath = \Yii::$app->getRuntimePath();
+        @unlink($runtimePath . '/rbac-test-items.php');
+        @unlink($runtimePath . '/rbac-test-assignments.php');
+        @unlink($runtimePath . '/rbac-test-rules.php');
+        $this->manager = new ExposedPhpManager([
+            'itemFile' => $runtimePath . '/rbac-test-items.php',
+            'assignmentFile' => $runtimePath . '/rbac-test-assignments.php',
+            'ruleFile' => $runtimePath . '/rbac-test-rules.php',
+        ]);
+    }
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 34. (undefined)

(ErrorControlOperator)